src/commands/postgres

```diff
@@ -30,7 +30,7 @@ env | grep "BAGCLI_*" | grep -v "PASS"
 
 postgresDump() {
   cli_log "Executing pg_dump"
-  pg_dump --format=c --no-owner --file="$BAGCLI_BACKUP_PATH"/backup.sqlc
+  pg_dump --format=c --no-owner --file="$BAGCLI_BACKUP_PATH"/backup.dump
```

Comment:
@qdupuy the extension is not a mistake. Is to help us recognize file format based on pgdump options.

Reply:
I put dump because this extension refers to the pg_dump command

While the extension you want to put back is confusing. We think you are talking about Structured Query Language when in fact you are just dumping with the custom format.